lib/torque/leaflet/canvas_layer.js

```diff
@@ -60,11 +60,11 @@ L.CanvasLayer = L.Class.extend({
     return canvas;
   },
 
-  onAdd: function (map) {
-    this._map = map;
+  _layerAdd: function (e) {
```

Comment:
Why can't we use `onAdd` here? `_layerAdd` is a private method that might change in the future, so I'd try to find a better solution for this.

There's also the `onRemove` in this file. We should make sure it's working properly.